cardcomm/pkcs11/src/common/libtomcrypt/tomcrypt_macros.h

```diff
@@ -212,7 +212,7 @@ asm("roll %%cl,%0": "=r"(word):"0"(word), "c"(i));
 	return word;
 }
 
-static inline unsigned ROR(unsigned word, int i)
+static inline unsigned ROR(unsigned word, long i)
```

Comment:
No. You can't just change the type of a parameter to an asm method without causing problems.